docs/usage/control-plane-endpoints-and-ports.md

```diff
@@ -0,0 +1,46 @@
+# Endpoints and Ports of a Shoot Control-Plane
+
+With the reversed VPN tunnel, there are no endpoints with open ports in the shoot cluster required by gardener.
+In order to allow communication to the shoots control-plane in the seed cluster, there are shared endpoints for all shoots of a seed cluster.
+The main entry-point into the seed cluster is the loadbalancer of the Istio ingress-gateway service. Depending on the infrastructure provider, there can be one IP address per zone.
+
+The Istio ingress-gateway exposes the following TCP ports:
+
+* **443** for requests to the shoot API-server. The request is dispatched according to the set TLS SNI extension.
+* **8443** for requests to the shoot API-server via `api-server-proxy`. 
```

Comment:
You could add that these requests are dispatched based to the proxy protocol target, which is the IP address of `kubernetes.default.svc.cluster.local` in the shoot.